src/test/java/com/google/api/codegen/protoannotations/testdata/csharp_library.baseline

```diff
@@ -48,20 +48,11 @@ namespace Google.Example.Library.V1.SmokeTests
     {
         public static int Main(string[] args)
         {
-            // Read projectId from args
-            if (args.Length != 1)
-            {
-                Console.WriteLine("Usage: Project ID must be passed as first argument.");
-                Console.WriteLine();
-                return 1;
-            }
-            string projectId = args[0];
-
             // Create client
             LibraryServiceClient client = LibraryServiceClient.Create();
 
             // Initialize request argument(s)
-            BookName name = new BookName($"testShelf-{Guid.NewGuid()}", projectId);
+            BookNameOneof name = BookNameOneof.From(new BookName("[BOOK_SHELF]", "[BOOK]"));
```

Comment:
I think that changing this type to `BookNameOneof` will break the C# client, so we should do field_name_patterns map parsing after this PR. Just filed https://github.com/googleapis/gapic-generator/issues/2734

Reply:
Cool, thanks!